ui/js/controllers/logviewer.js

```diff
@@ -268,11 +268,15 @@ logViewerApp.controller('LogviewerCtrl', [
 
                             if ($scope.step_data.all_errors.length == 0) {
                                 angular.element(document).ready(function () {
-                                    if (isNaN($scope.selectedBegin)) {
```

Comment:
I think this ``isNaN($scope.selectedBegin...`` is necessary if the user has passed in a line number on the URL.  If they have, then that line number should be used.  Can you test this with the ``#L<num>`` urls?  (like the links you created in your last PR) ?  It looks to me like this will ignore that value and go to the first non-success no matter what.